infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/RoutingQueuesLogEntryReader.java

```diff
@@ -25,8 +35,29 @@ public RoutingQueuesLogEntryReader(CorfuRuntime runtime, LogReplicationSession s
 
     @Override
     public LogReplicationEntryMsg read(UUID logEntryRequestId) throws TrimmedException {
-        // Reads from the queue corresponding to this destination.  The queue contains addresses in the main queue
-        // from where the actual data(payload) is to be read.
```

Comment:
I think method should be removed instead of a no-op, as we are still using the `read` logic in base class